keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -77,6 +79,21 @@ File and block are first class citizens within the Kubernetes ecosystem.  Object
 
 # Proposal
 
+## User Stories
+
+#### Admin
+
+- As a cluster administrator, I can set generated buckets' storage capacity via the Kubernete's API, so that workloads may not inordinately contribute to monthly infrastructure costs.
+- As a cluster administrator, I can use Kubernetes RBAC policies around bucket APIs, so that I may control access to existing buckets from within the cluster, reducing the need to adminster an external storage interface.
+- As a cluster administrator, I can manage multiple object store providers via the Kubernetes interface, so that I do not have to become an expert in multiple, drastically different storage interfaces.
+
+#### User
```

Comment:
Does it make sense to have a `Provisioner` persona too?  E.g
_As an object store vendor, I want to develop a bucket provisioning plugin leveraging CSI knowledge in my team.

Reply:
I'm not entire sure.  To me, defining the admin and user stories helps to get a vision of the system that storage vendors would help build.  So they're half user, half developer.